composer.json

```
@@ -35,5 +35,8 @@
     },
     "bin": [
         "parallel-lint"
-    ]
+    ],
+    "scripts": {
+        "tests": "@php vendor/bin/tester -p php tests"
```

```suggestion
        "test": "@php vendor/bin/tester -p php tests"
```
It would be more consistent to use "test" as a verb, like "install", "update", etc.

---

FTR the existing one was already named `tests`